script/build.js

```diff
@@ -168,7 +168,9 @@ if (options.watch) {
     contentBase: `build/${options.buildtype}`,
     historyApiFallback: {
       rewrites: [
-        { from: '^\/rx(.*)', to: '/rx/' }
+        { from: '^\/rx(.*)', to: '/rx/' },
+        { from: '^\/healthcare\/apply\/application(.*)', to: '/healthcare/apply/application/' },
```

Comment:
Education benefits is using education/apply-for-education-benefits/apply as the entry page right now.

`{ from: '^\/education\/apply-for-education-benefits\/apply(.*)', to: '/education/apply-for-education-benefits/apply/' },`, maybe?